### PR TITLE
update pathfinder

### PR DESCRIPTION
**Pull request by JacqueGM**
Core test updated to test csv export and `--flowcsv` tag in `circles-api` with this PR https://github.com/CirclesUBI/circles-api/pull/123

there is a docker image built [v1.6.0-51-g5f29e1f](https://github.com/CirclesUBI/circles-api/tree/v1.6.0-51-g5f29e1f)

use circles-docker  branch --> [`update-pf`](https://github.com/CirclesUBI/circles-docker/tree/update-pf)
  - https://github.com/CirclesUBI/circles-docker/pull/52

### CODE REVIEW

File: test/token.test.js [hunk 1]
```diff
@@ -338,7 +305,7 @@ describe('Token', () => {
         core.token.transfer(accounts[0], {
           from: safeAddresses[0],
           to: safeAddresses[4],
-          value: web3.utils.toBN(core.utils.toFreckles('5')),
+          value: web3.utils.toBN(core.utils.toFreckles('100000000')),
```
Comment (llunaCreixent) [inline test/token.test.js:308]: It didn't pass with 5 freckles? 
This test is made for testing a transfer with a quantity (5) which fails at the beginning but after updating, succeeds.
If it doesn't work with a 5, we have to review the test more in detail.

File: test/token.test.js [hunk 2]
```diff
@@ -309,25 +288,13 @@ describe('Token', () => {
...
-          to: safeAddresses[4],
-          token: safeAddresses[0],
-          capacity: '100',
-        });
-        // Add backslashes to scape the double quote symbol
-        const edgesDataString = stringify(edgesData).replace(/"/g, '\\"');
+        let edgesCSVdata = `${safeAddresses[0]},${safeAddresses[4]},${safeAddresses[0]},100`;
         execSync(
-          `docker exec circles-api bash -c "echo '${edgesDataString}' > edges-data/edges.json"`,
+          `docker exec circles-api bash -c "echo '${edgesCSVdata}' >> edges-data/edges.csv" `,
```
Comment (llunaCreixent) [inline test/token.test.js:297]: Would make sense to add the new row with the `fs` library?

File: test/token.test.js [hunk 3]
```diff
@@ -309,27 +287,16 @@ describe('Token', () => {
...
-        );
-        edgesData.push({
-          from: safeAddresses[0],
-          to: safeAddresses[4],
-          token: safeAddresses[0],
-          capacity: '100',
-        });
-        // Add backslashes to scape the double quote symbol
-        const edgesDataString = stringify(edgesData).replace(/"/g, '\\"');
+        let edgesCSVdata = `${safeAddresses[0]},${safeAddresses[4]},${safeAddresses[0]},100000000000000000000`;
```
Comment (llunaCreixent) [inline test/token.test.js:294]: I feel I had implemented the test incorrectly before... mmmm 🤔

**Comment by llunaCreixent** [comment]
I think all is good now :) 
But please test and review

File: test/token.test.js (hunk 3 above)
Comment (llunaCreixent) [reply test/token.test.js:294]: can you check @JacqueGM that the test makes sense?

**Review by llunaCreixent — DISMISSED** [review]

File: test/token.test.js (hunk 3 above)
Comment (JacqueGM) [reply test/token.test.js:294]: test pass on my local now ^^

**Review by llunaCreixent — APPROVED** [review]
🐶

**Pull request merged**